e2e_samples/parking_sensors/adf/linkedService/Ls_AzureDatabricks_01.json

```diff
@@ -1,20 +1,21 @@
 {
-	"name": "Ls_AzureDatabricks_01",
-	"properties": {
-		"annotations": [],
-		"type": "AzureDatabricks",
-		"typeProperties": {
-			"domain": "https://adb-7988601177969121.1.azuredatabricks.net",
-			"authentication": "MSI",
-			"workspaceResourceId": "/subscriptions/c20af4c3-6b7a-4ee8-b830-a2a69c4a56ac/resourceGroups/mdwdops-16-dev-rg/providers/Microsoft.Databricks/workspaces/mdwdops-dbw-dev-16",
-			"newClusterNodeType": "Standard_DS3_v2",
-			"newClusterNumOfWorker": "1",
-			"newClusterSparkEnvVars": {
-				"PYSPARK_PYTHON": "/databricks/python3/bin/python3"
-			},
-			"newClusterVersion": "10.4.x-scala2.12",
-			"newClusterInitScripts": []
-		}
-	},
-	"type": "Microsoft.DataFactory/factories/linkedservices"
+    "name": "Ls_AzureDatabricks_01",
+    "properties": {
+        "annotations": [],
+        "type": "AzureDatabricks",
+        "typeProperties": {
+            "domain": "https://adb-1380542669290933.13.azuredatabricks.net",
+            "authentication": "MSI",
+            "workspaceResourceId": "/subscriptions/334f902c-99ae-4637-9967-767272b6922c/resourceGroups/mdwdops-ys08-dev-rg/providers/Microsoft.Databricks/workspaces/mdwdops-dbw-dev-ys08",
+            "newClusterNodeType": "Standard_DC4as_v5",
```

Comment:
please don't use ACC VMs by default. I think this will be solved if you fix the `cluster.config.json` file though.